WcaOnRails/config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
```

Comment:
Oh yes this is the second thing I wanted to comment on : I intentionally did not indent the routes in scope to make an potential rebase easier.
They should obviously be indented in the end :)